index.html

```diff
@@ -185,12 +185,14 @@ <h2 class="major">ML showcase</h2>
                 <a href="#" class="image"
                   ><img src="images/rm/bigney.jpg" alt=""
                 /></a>
-                <h3 class="major">Predicting chronic kidney disease</h3>
+                <h3 class="major">
+                  Prediciendo la enfermedad crónica de rinión

```

Comment:
Nit: te regalo una ñ 😁
```suggestion
                  Prediciendo la enfermedad crónica de riñón
```